proto/neutron/state_verifier/v1/query.proto

```diff
@@ -7,21 +7,22 @@ import "neutron/interchainqueries/tx.proto";
 option go_package = "github.com/neutron-org/neutron/v5/x/state-verifier/types";
 
 service Query {
-  // VerifyStateValues verifies the values and returns an error if values cannot be verified. The query returns `QueryVerifyStateValuesResponse` response.
+  // Verifies the values and returns an error if values cannot be verified. The query returns `QueryVerifyStateValuesResponse` response.
   rpc VerifyStateValues(QueryVerifyStateValuesRequest) returns (QueryVerifyStateValuesResponse) {
     option (google.api.http).get = "/neutron/state-verifier/verify_state_values";
   }
 }
 
-// QueryVerifyStateValuesRequest describes a structure to verify storage values from Neutron state from a particular height in the past
+// Describes a structure to verify storage values from the chain state from a particular height in the past
 message QueryVerifyStateValuesRequest {
-  // height refers to the block height to which the storage values belong.
+  // Refers to the block height to which the storage values belong.
   uint64 height = 1;
-  // storage_values that are present on the specified height and must be verified against
+  // A slice of []StorageValue that are present on the specified height and must be verified against
```

Comment:
1. `A slice of []StorageValue` makes it a slice of slices which is incorrect and might be confusing
2. `that are present on the specified height` is also incorrect because there might be non-existence proofs provided
3. the `StorageValue` type is imported from the icq module which doesn't seem to be correct: it doesn't have to do with the icq module. do you have ideas on how to do it the right way?

Reply:
1. Fixed
2. Fixed
3. You are implying that either we need to move this type into state-verifier module, or maybe even into a separate utils package (where we already values verification code). It make sense but:
    1. Our current project structure does not allow us to generate proto code in some custom folder/package (e.g. /utils). It's possible to do that only into x/<MODULE>. We can consider adding this functionality but it's out of scope.
    2.  I don't like the idea of moving this type somewhere else from the ICQ module because it's not kinda backwards compatible. I'll try to think about what can be broken exactly but let's not do this right now anyway.